serl_robot_infra/franka_env/spacemouse/spacemouse_test.py

```diff
@@ -0,0 +1,33 @@
+""" Test the spacemouse output. """
+
+
+def test_spacemouse():
+    """Test the SpaceMouseExpert class.
+
+    This interactive test prints the action and buttons of the spacemouse at a rate of 10Hz.
+    The user is expected to move the spacemouse and press its buttons while the test is running.
+    It keeps running until the user stops it.
+
+    """
+    import time
+    import numpy as np
```

Comment:
import should be on top of the file

Reply:
I think I misunderstood then. You want imports to be on top, but the setting of `numpy` precision options to be in the test? both of these are global changes.

Reply:
import should always go to the top of a py file,

np print precesion is local to your function, e.g., you can change in another function; so it should go to the function which actually needs it

Reply:
np print precision is not local. But we can make it local with a context manager. just pushed a change